routes/employee.js

```diff
@@ -3,8 +3,8 @@ const controller = require('../controllers/employee')
 const router = express.Router()
 
 router.get('/:id', controller.getById)
-router.delete('/:id', controller.deleteById)
-router.post('/department/id', controller.create)
+router.delete('/:id/department/:departmentId', controller.deleteById)
```

Comment:
you need both options:
delete employee by employee id 
and delete employee from department